media/search.js

```diff
@@ -0,0 +1,66 @@
+//@ts-check
+
+// This script will be run within the webview itself
```

Comment:
this file has just been renamed and formatted - it is the js behind the rule search functionality